### PR TITLE
Created 2427.c

### PR DESCRIPTION
**Pull request by anuragsahay5**
#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)
- [x] Added tests and example, test must pass
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

File: leetcode/src/2427.c [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (Panquesito7) [inline leetcode/src/2427.c:1]: Please add small documentation explaining the problem.
You can check other files as an example of the same. 🙂

You also need to add this new explanation to the [list of LeetCode problems](https://github.com/TheAlgorithms/C/blob/master/leetcode/README.md).

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: leetcode/src/2427.c (hunk 1 above)
Comment (anuragsahay5) [reply leetcode/src/2427.c:1]: Ok

**Comment by Panquesito7** [comment]
Do you need any help with this, @anuragsahay5?

**Comment by anuragsahay5** [comment]
> Do you need any help with this, @anuragsahay5?

**Pull request closed without merging**